### ISSUE TITLE
Accessibility web developper tool

### ISSUE THREAD

**Issue by erynjones** [erynjones/dsn1675-saas-website#6]
I was running my website threw the google chrome accessibility web developer tool and some of the things that turned out weren't all green and i am not sure what to do about the red sections. 
@thomasjbradley
<img width="1440" alt="screen shot 2015-11-22 at 6 45 28 pm" src="https://cloud.githubusercontent.com/assets/10467665/11327093/cc792410-9149-11e5-9069-52993d6b044c.png">
<img width="1439" alt="screen shot 2015-11-22 at 6 46 17 pm" src="https://cloud.githubusercontent.com/assets/10467665/11327094/cc7b8a52-9149-11e5-8885-85c12dd8a06c.png">
<img width="1440" alt="screen shot 2015-11-22 at 6 46 43 pm" src="https://cloud.githubusercontent.com/assets/10467665/11327095/cc7d2506-9149-11e5-9d91-f7a27984736b.png">
<img width="1439" alt="screen shot 2015-11-22 at 6 47 33 pm" src="https://cloud.githubusercontent.com/assets/10467665/11327096/cc7f38be-9149-11e5-80bd-a913e6f831aa.png">

**Comment by thomasjbradley** [comment]
So there are a few that should be fixed and a few that you can ignore.

_Ignorable:_
- "[Warning] These elements are focusable but either invisible…"
- "[Warning] Text elements should have reasonable contrast ratio." (But this could be caused when images are disabled if the text becomes invisible)

_Fixable:_
- "[Severe] An element's ID must be unique…" — the `id=""` attribute on your `<input>` tags cannot be repeated. For instance in the signup form there's an `id="first"` then it's repeat again in the credit card form. Change one of them to something like `id="cc_first"` for "credit card first". *Make sure to also change the matching `<label for="">`.
- "[Severe] Elements with ARIA roles must use a valid…" — there's a typo in "contentinfo".
- "[Warning] This element has an invalid ARIA attribute…" — looks like a typo in "label".
- "[Severe] Controls and media elements should have labels" — there's a typo with the IDs on these two lines: https://github.com/erynjones/dsn1675-saas-website/blob/gh-pages/sign-up.html#L103-L104
